iidm/iidm-xml-converter/src/main/java/com/powsybl/iidm/xml/NetworkXml.java

```diff
@@ -354,6 +354,7 @@ public static Anonymizer write(Network n, ExportOptions options, OutputStream os
             writeExtensions(n, context, options);
             context.getWriter().writeEndElement();
             context.getWriter().writeEndDocument();
+            context.getWriter().flush();
```

Comment:
Are you sure you need this, I cannot see how it is related to the encoding config.

Reply:
it seems that encoding config are cached data and we need to flush after the end in order to write those data.

removing this line, generate test failure when encoding is not set as default